### ISSUE TITLE
Retroarch cheats

### ISSUE THREAD

**Issue by CitizenGit** [libretro/libretro-database#1470]
Hi, I would like to start off by saying great job. I have some questions about how to use the cht codes in Resident Evil 2 for the dreamcast via Retroarch. What exactly does it mean to "Replace FF's with below code?"

Thank you in advance.

**Comment by OctopusButtons** [comment]
Yeah it's undocumented sadly. That cheat data was added by [this contributor Pull Request here](https://github.com/libretro/libretro-database/pull/710).  They scraped it from somewhere, so either the original was equally confusing or something was lost.

Sometimes "replace" codes mean you replace a certain string with the code, like these "??" codes here: https://www.almarsguides.com/retro/walkthroughs/Dreamcast/Games/ResidentEvil2/CodeBreaker/ChestItemModifierCodes/  They didn't document it either, but I assume you replace the ?? with a 2-digit quantity, and that's the quantity that will be forced to whatever is in the given slot.

If you're interested, you might be uniquely suited to fix the problem for everyone.  If you figure out what it means or how to use it, you can then fork the repository in github, click pencil button to EDIT your copy of the repository, add some meaningful instructions/descriptions into just the first description for slot 1, then "Commit" with a Pull Request to libretro.  Then the maintainers will probably accept your helpful changes in, and everyone will see your new helpful description when they use/update the same cheat.

Also based on the Almars Guide cheats I linked above, you could probably make your own easier `.cht` file more like this:

```
cheat0_desc = "Slot 1 item quantity 10"
cheat0_code = "00DD95A0+00000010"
cheat0_enable = false
```

Or replace the 10 with whatever.  And then you could add your personal .cht file to the repository by the same process I mentioned above.
